src/main/java/com/treasuredata/client/TDClientConfig.java

```diff
@@ -53,6 +53,7 @@
         USESSL("td.client.usessl", "Use SSL encryption"),
         API_ENDPOINT("td.client.endpoint", "TD API end point (e.g., api.treasuredata.com"),
         API_PORT("td.client.port", "TD API port number"),
+        RETRY_STRATEGY("td.client.retry.strategy", "The retry strategy for API request"),
```

Comment:
We don't need to use string-based properties anymore. These configurations were implemented for compatibility reasons, but we can deprecate this usage.